src/views/Login.js

```diff
@@ -0,0 +1,9 @@
+import React, { Component } from "react";
+
```

Comment:
```suggestion
const LoginComponent = () => (
  <h1 style={{ background: 'blue', color: 'white' }}>Login Component</h1>
);
```

Refactor to this. When using a functional component that has a single return value you do not need the `{ }` or the `return statement`. The `( )` automatically resolve to a return.